docs/validators/validator-setup.md

```diff
@@ -1,15 +1,20 @@
 # Run a Validator on Desmos
-If you plan to use a KMS (key management system), you should go through these steps first: [Using a KMS](kms/kms.md).
 
 ## What is a Validator?
+
 [Validators](overview.md) are responsible for committing new blocks to the blockchain through voting. A validator's stake is slashed if they become unavailable or sign blocks at the same height. Please read about [Sentry Node Architecture](validator-faq.md#how-can-validators-protect-themselves-from-denial-of-service-attacks) to protect your node from DDOS attacks and to ensure high-availability.
 
 ::: danger Warning
 If you want to become a validator for the `mainnet`, you should [research security](security.md).
 :::
 
+## Running a Fullnode
+
+To become a validator, you must first have `desmosd` and `desmoscli` installed and be able to run a fullnode. You can first [setup your fullnode](/fullnode/overview.html) if you haven't yet.
```

Comment:
I think it's better to change the link to the MD file to make it easier to navigate to even from the repo and IDEs. 

```suggestion
To become a validator, you must first have `desmosd` and `desmoscli` installed and be able to run a fullnode. You can first [setup your fullnode](/fullnode/overview.md) if you haven't yet.
```